### PR TITLE
Adding tasks for clinical validation.

### PR DESCRIPTION
**Pull request by rhpvorderman**
### Checklist
- [x] Pull request details were added to CHANGELOG.md

### CODE REVIEW

File: gatk.wdl [hunk 1]
```diff
@@ -1347,6 +1347,63 @@ task PreprocessIntervals {
...
+
+    parameter_meta {
+        inputVcf: {description: "The VCF input file.", category: "required"}
+        inputVcfIndex: {description: "The input VCF file's index.", category: "required"}
+        referenceFasta: {description: "The reference fasta file which was also used for mapping.",
+                         category: "required"}
+        referenceFastaDict: {description: "The sequence dictionary associated with the reference fasta file.",
+                             category: "required"}
+        referenceFastaFai: {description: "The index for the reference fasta file.", category: "required"}
+        selectTypeToInclude: {description: "Select only a certain type of variants from the input file", category: "common"}
```
Comment (DavyCats) [inline gatk.wdl]: Is this a common input? I seems more like an advanced one to me.

File: gatk.wdl [hunk 2]
```diff
@@ -1347,6 +1347,63 @@ task PreprocessIntervals {
...
+        outputPath: {description: "The location the output VCF file should be written.", category: "required"}
+        intervals: {description: "Bed files or interval lists describing the regions to operate on.", category: "advanced"}
```
Comment (DavyCats) [inline gatk.wdl]: Intervals seem like they would be more common than selectTypeToInclude.

File: gatk.wdl [hunk 3]
```diff
@@ -1401,3 +1458,62 @@ task SplitNCigarReads {
...
+
+task VariantFiltration {
+    input {
+        File inputVcf
+        File inputVcfIndex
+        File referenceFasta
+        File referenceFastaDict
+        File referenceFastaFai
+        String outputPath = "filtered.vcf.gz"
+        Array[String]+ filterArguments
```
Comment (DavyCats) [inline gatk.wdl]: Hmm, I don't really like this. Is this always a combination of `filter_name` and `filter_expression`, maybe in that case a Pair could be used in some way?

File: rtg.wdl [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+    runtime {
+        docker: dockerImage
+        memory: memory
+    }
+
+    parameter_meta {
+        format: {description: "Format of input. Allowed values are [fasta, fastq, fastq-interleaved, sam-se, sam-pe] (Default is fasta)",
+                 category: "advanced"}
+        outputPath: {description: "Where the output should be placed.", category: "advanced"}
+        inputFiles: {description: "input sequence files. May be specified 1 or more times."}
```
Comment (DavyCats) [inline rtg.wdl]: missing category

File: gatk.wdl (hunk 3 above)
Comment (rhpvorderman) [reply gatk.wdl]: On the command line you would need to do this: `--filter-name my_filter --filter-expression 'GQ<30' --filter-name my_other_filter --filter-expression 'AD>2.0'`

So I reckoned that a list is the easiest way: `["--filter-name", "my_filter", "--filter-expression", "'GQ<30'", "--filter-name", "my_other_filter", "--filter-expression", "'AD>2.0'"]`

But maybe a string is even easier:
`"--filter-name my_filter --filter-expression 'GQ<30' --filter-name my_other_filter --filter-expression 'AD>2.0'"`

Pairs are difficult to deserialize in tasks + they are a pain to use as input in JSON. This is a list of pairs... 

What do you think?

File: gatk.wdl (hunk 2 above)
Comment (rhpvorderman) [reply gatk.wdl]: Well SelectTypeToInclude is also quite common, but I agree, I will make intervals common as well.

File: gatk.wdl (hunk 1 above)
Comment (rhpvorderman) [reply gatk.wdl]: This is used in the clinical evaluation pipeline. It is very useful to distinguish between SNPs and indels when evaluating.

File: gatk.wdl [hunk 5]
```diff
@@ -1347,6 +1347,63 @@ task PreprocessIntervals {
...
+    parameter_meta {
+        inputVcf: {description: "The VCF input file.", category: "required"}
+        inputVcfIndex: {description: "The input VCF file's index.", category: "required"}
+        referenceFasta: {description: "The reference fasta file which was also used for mapping.",
+                         category: "required"}
+        referenceFastaDict: {description: "The sequence dictionary associated with the reference fasta file.",
+                             category: "required"}
+        referenceFastaFai: {description: "The index for the reference fasta file.", category: "required"}
+        selectTypeToInclude: {description: "Select only a certain type of variants from the input file", category: "common"}
+        outputPath: {description: "The location the output VCF file should be written.", category: "required"}
```
[the remark this answers is not in the document]
Comment (rhpvorderman) [reply gatk.wdl]: :+1: I will make this advanced. People should not be messing with this really.

**Pull request merged**